lib/view_models/page_view_model/join_organization_viewModel.dart

```diff
@@ -179,23 +178,20 @@ class JoinOrgnizationViewModel extends BaseModel {
     getCurrentUserId();
     //function to fetch the org from the server
     final GraphQLClient _client = graphQLConfiguration.authClient();
-
-    final QueryResult result = await _client
+    final QueryResult organizationQueryResult = await _client
         .query(QueryOptions(documentNode: gql(_query.fetchOrganizations)));
-
     // Get the details of the current user.
     final QueryResult userDetailsResult = await _client.query(QueryOptions(
         documentNode: gql(_query.fetchUserInfo),
         variables: {'id': _currentUserId}));
 
-    if (result.hasException || userDetailsResult.hasException) {
-      print(result.exception);
-      showError(result.exception.toString());
-    } else if (!result.hasException &&
+    if (organizationQueryResult.hasException ||
```

Comment:
Could you update this similar to line 222. just makes it consistent and easy to read.

Reply:
Sir, actually we are not categorizing the type of exception in org fetching or user fetching.
And also we are not doing anything like "getNewToken" in this part, we are just using bool value from the backend(.hasException).
If there are any categorising in the future I'll add that particular enum.
Ps: I've fixed the spelling error.

Reply:
I had to create a separate toast container file (code was earlier in the file "SuccessToast .dart" which was deleted in recent merge).